rules/windows/execution_unusual_dns_service_file_writes.toml

```
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2020/07/16"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/16"
+
+[rule]
+author = ["Elastic"]
+description = "An unexpected file was modified by dns.exe, which is responsible for DNS Server services."
+index = ["winlogbeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Unusual File Modification by dns.exe"
+references = [
+    "https://research.checkpoint.com/2020/resolving-your-way-into-domain-admin-exploiting-a-17-year-old-bug-in-windows-dns-servers/",
+    "https://msrc-blog.microsoft.com/2020/07/14/july-2020-security-update-cve-2020-1350-vulnerability-in-windows-domain-name-system-dns-server/",
+]
+risk_score = 73
+rule_id = "c7ce36c0-32ff-4f9a-bfc2-dcb242bf99f9"
+severity = "high"
+tags = ["CVE-2020-1350", "Elastic", "Windows"]
```

I believe these tags populate in the UI. Do we want to have this specific CVE tag in the UI as opposed to just the standard “Windows” tag.

---

👋 

Yep, I think so. Since we tend to normally target behavior rather than specific vulnerabilities, rules specific to a CVE will be less frequent. In cases where the CVE is a 10 and has significant impact, if it makes sense to have a rule, I think it makes sense to tag the CVE it may be more commonly known by.

---

I don't think we should treat tags as one-offs, but more as ways to build medium-large groups of rules.

I think we should instead support more threat frameworks than just ATT&CK

```suggestion
tags = ["Elastic", "Windows"]
```

---

I agree. I think it could get a bit messy and dated over an extended period of time to have one off tags, unless we had a better way to sort or visualize tags in the UI.

---

We had a discussion and have decided to remove any mention of CVE from tags altogether. We still have to come to a decision on when we want to associate a CVE to a rule  in general (_only rules specific to a CVE, only CVE 10, etc._), but once we figure that it, it would most likely be better fit within the `threat` array, such as

```json
threat = {
    "framework": "CVE",
    "ID": "2020-1350",
    "CVSS": 10
}
```

+- any other relevant fields we would want to build into that structure. The API would need updating to support this first as well (cc @spong)